### PR TITLE
refactor: some more profile changes

### PR DESCRIPTION
**Pull request by nchamo**
Some more minor changes

### CODE REVIEW

File: kernel/packages/shared/dao/reducer.ts [hunk 1]
```diff
@@ -122,7 +121,7 @@ function realmProperties(realm: Realm, configOverride: boolean = true): Partial<
   return {
     profileServer: FETCH_PROFILE_SERVICE && configOverride ? FETCH_PROFILE_SERVICE : domain + '/lambdas/profile',
     fetchContentServer: FETCH_CONTENT_SERVICE && configOverride ? FETCH_CONTENT_SERVICE : domain + '/lambdas/contentv2',
-    fetchMetaContentServer: FETCH_META_CONTENT_SERVICE && configOverride ? FETCH_META_CONTENT_SERVICE : domain,
+    catalystServer: domain,
```
Comment (pablitar) [inline kernel/packages/shared/dao/reducer.ts:121]: Aren't we removing functionality here?

File: kernel/packages/shared/dao/reducer.ts [hunk 2]
```diff
@@ -122,7 +121,7 @@ function realmProperties(realm: Realm, configOverride: boolean = true): Partial<
   return {
     profileServer: FETCH_PROFILE_SERVICE && configOverride ? FETCH_PROFILE_SERVICE : domain + '/lambdas/profile',
```
Comment (pablitar) [inline kernel/packages/shared/dao/reducer.ts]: If I understood correctly, we are replacing this. Can we delete it?

**Review by pablitar — DISMISSED** [review]
I'm like simplifying our configuration.

Just in case, please consider the comments :)

File: kernel/packages/shared/dao/reducer.ts (hunk 1 above)
Comment (nchamo) [reply kernel/packages/shared/dao/reducer.ts:121]: Yeah, indeed. It's not overridable anymore, but I don't think it's being used. If we think we need it back, we can add it back in the future

File: kernel/packages/shared/dao/reducer.ts (hunk 2 above)
Comment (nchamo) [reply kernel/packages/shared/dao/reducer.ts]: Good catch, deleting it

**Review by pablitar — APPROVED** [review]
LGTM!

**Pull request merged**